frontend/src/app/main/ui/workspace/tokens/management/forms/controls/input.cljs

```diff
@@ -39,7 +39,7 @@
                 (rx/of {:value resolved-value})
                 (rx/of {:error (first errors)}))))))))
 
-(mf/defc input-token*
+(mf/defc input*
```

Comment:
I'll give this a more specific name, since this component is not a mere input, but an input that edits the `:value` property of a simple token.

For example: `token-value-input`. And I would add a docstring stating this.

Similarly, the `color-input` could be `color-token-value-input`. We should not be afraid of long names where necessary.